multi_import/import_diff_applier.py

```diff
@@ -25,27 +28,31 @@ def get_object_changes(self, attributes, obj, update=False):
             if not value:
                 continue
 
-            mapping = self.mappings.fields[attribute]
+            field = self.serializer.fields.get(column_name, None)
+
+            if not field:
+                continue
 
-            if mapping.readonly:
+            if field.read_only:
                 continue
 
-            if mapping.is_relationship:
+            if isinstance(field, RelatedField):
                 # TODO: Add support for new object refs
                 if num_values != required_values:
                     continue
+                if value:
+                    value = field.related_model.objects.get(pk=value)
 
-                if mapping.is_foreign_key:
-                    if value:
-                        value = mapping.related_model.objects.get(pk=value)
-
-                if mapping.is_one_to_many:
-                    value = [
-                        mapping.related_model.objects.get(pk=val)
-                        for val in value
-                    ]
+            if isinstance(field, ManyRelatedField):
+                # TODO: Add support for new object refs
+                if num_values != required_values:
+                    continue
+                value = [
+                    field.child_relation.related_model.objects.get(pk=val)
+                    for val in value
```

Comment:
This should probably be something like:

    value = list(
        field.child_relation.related_model.objects.filter(pk__in=value)
    )

to avoid the query per-item thing here.